crates/rpc-jsonrpsee/src/traits.rs

```diff
@@ -135,3 +135,13 @@ pub trait EngineApiExt {
     #[method(name = "signalSuperchainV1")]
     async fn signal_superchain_v1(&self, signal: SuperchainSignal) -> RpcResult<ProtocolVersion>;
 }
+
+/// Op API extension for controlling the miner.
+#[cfg_attr(not(feature = "client"), rpc(server, namespace = "miner"))]
+#[cfg_attr(feature = "client", rpc(server, client, namespace = "miner"))]
```

Comment:
Curious, is it destructive to do the following instead?
```rust
#[rpc(server, namespace = "miner")]
#[cfg_attr(feature = "client", rpc(client))]
```

Reply:
iirc this didn't quite work, but can't remember specifics